web/packages/teleport/src/Discover/SelectResource/types.ts

```diff
@@ -127,6 +132,7 @@ export interface ResourceSpec {
    * An empty array or undefined means that the resource supports all auth types.
    */
   supportedAuthTypes?: AuthType[];
+  id: DiscoverGuideId;
```

Comment:
curious, is is deliberate that this new field added as last item but another new field `pinned?: boolean;` added as first item?

Reply:
not deliberate 😅 , i updated to put both fields at top since it's more fitting